test/e2e/secret_test.go

```diff
@@ -29,6 +29,13 @@ var _ = Describe("Podman secret", func() {
 		secrID := session.OutputToString()
 		Expect(session).Should(ExitCleanly())
 
+		result := podmanTest.Podman([]string{"events", "--stream=false"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(ExitCleanly())
+		events := result.OutputToStringArray()
+		Expect(events).ToNot(BeEmpty(), "Number of events")
+		Expect(events).To(ContainElement(ContainSubstring(fmt.Sprintf(" secret create %s", secrID))))
```

Comment:
```suggestion
		Expect(events).To(ContainElement(ContainSubstring(" secret create %s", secrID)))
```
ContainSubstring() already handled the sprintf call when extra args are given.

Reply:
Ah, didn't know that. I will improve it.